StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1008OpeningParenthesisMustBeSpacedCorrectly.cs

```diff
@@ -100,6 +100,9 @@ private static void HandleOpenParenToken(SyntaxTreeAnalysisContext context, Synt
             bool precededByKeyword;
             switch (prevToken.Kind())
             {
+            case SyntaxKind.None:
+                return;
```

Comment:
📝 This won't correctly handle the following top-level program:

```cs
 ( var a, var b) = (3, 3);
System.Console.WriteLine($"{a}");
```

Reply:
Good catch! Didn't think of that.
Alright, I will have a look.

Reply:
I see now that you added both a leading and a trailing space in your example. What was your intention with the leading space?

Reply:
It should behave the same way as this:

```cs
void Method()
{
     ( var a, var b) = (3, 3);
    System.Console.WriteLine($"{a}");
}
```

Reply:
As far as I can understand, this is a "don't care" case, since it is the first token on the line